packages/wrangler/src/config/validation.ts

```diff
@@ -2212,18 +2212,26 @@ const validateCloudchamberConfig: ValidatorFn = (diagnostics, field, value) => {
 		return false;
 	}
 
+	const optionalAttrsByType = {
+		string: ["memory", "image", "location"],
+		boolean: ["ipv4"],
+		number: ["vcpu"],
+	};
+
 	let isValid = true;
-	const requiredKeys: string[] = [];
```

Comment:
Seems that this was doing nothing. Replaced with type checking on the optional params (all of them) & added a test.